gokart/run.py

```diff
@@ -118,5 +127,6 @@ def run(cmdline_args=None, set_retcode=True):
         event_aggregator.set_handlers()
         luigi.cmdline.luigi_run(cmdline_args)
     except SystemExit as e:
+        _dump_task_info(cmdline_args=cmdline_args, dump_task_info_path=dump_task_info_path)
```

Comment:
[ASK] Why implement it in such a way that you can change it when you do `run` it?

It could be an environment variable like `Slack API`.
Or, you can choose not to let the user choose like `other log files`.

Reply:
- I thought setting environment variable is difficult, since it may cause users to make mistake or forget at settings
- I thought allowing to change save path would be kind to users
- I thought setting path at `gokart.run()` is understandable, since the dumped file contains resulted information of `run()`

https://github.com/m3dev/gokart/pull/176#discussion_r574907617
As mentioned here, I think setting dump/no dump in the `cmdline_args` is better.

I also think that there should be default file path for convenience. 

Considering above, I will change the implementation as,
- set dump/no dump in the `cmdline_args`
- dump file path can be set
- when no path is specified, the file will be dumped to default path (e.g. workspace/task_info.csv)

How do you think? @vaaaaanquish @Hi-king

Reply:
> cmdline_args  

I feel not happy with anything that can't be managed in a `Config File`. It would be nice if we could pass the Config file.

> dump file path can be set  

LGTM

> when no path is specified, the file will be dumped to default path (e.g. workspace/task_info.csv)  

Sounds good. Something like `workspace/task_info/run_xxxxxxxxx.pkl`